lib/child_process.js

```diff
@@ -440,9 +440,11 @@ function normalizeSpawnArguments(file, args, options) {
     const command = [file].concat(args).join(' ');
 
     if (process.platform === 'win32') {
-      file = typeof options.shell === 'string' ? options.shell :
-              process.env.comspec || 'cmd.exe';
-      args = ['/d', '/s', '/c', '"' + command + '"'];
+      if (typeof options.shell === 'string')
+        file = options.shell;
+      else
+        file = process.env.comspec || 'cmd.exe';
```

Comment:
@refack No, it is not, at least not in the top-level environment. `%COMSPEC%` must always be a valid path to an executable acting as the "shell". However, the value of `%COMSPEC%` can be modified by users and can be overriden and/or deleted for child processes. Windows will not prevent that, even though it is not intended behavior.